test/test_looplibrary_environment.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+exit_code=0
+
+exe_test () {
+   TEST_LOOPLIBRARY=$(./test_looplibrary_environment.x \
```

Comment:
I think this executable is currently missing.

Reply:
0b867f0